extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -159,6 +159,17 @@ export class AmpSlideScroll extends BaseSlides {
       this.slidesContainer_.addEventListener(
           'touchend', this.touchEndHandler_.bind(this));
     }
+
+    this.registerAction('moveToSlideIndex', invocation => {
```

Comment:
IMO, `slide` is a more specific form of `index` and is closer to colloquial usage. E.g. when presenting a slide deck, one would say "go to the next slide". 

`goToSlide` works in this form but "on: go to slide" doesn't read naturally. For example:

`<amp-carousel type=slides on="goToSlide:otherCarousel.goToSlide(index=3)">`

vs.

`<amp-carousel type=slides on="changeSlide:otherCarousel.changeSlide(slide=3)">`

This is nitpicky though and I don't really mind. Feel free to get a PM to tiebreak or ACK this comment. :)